src/scss/components/_media-element.scss

```diff
@@ -158,3 +158,10 @@ $mediaElement-spacing           : spacing(x2) !default;
             text-decoration: underline;
         }
     }
+
+    .c-mediaElement-centeredLink {
```

Comment:
Yeah, looking back at this maybe it's not the best way of handling this part of the layout. 

What I am looking to achieve is the layout circled here:

<img width="233" alt="screen shot 2018-08-23 at 10 05 58" src="https://user-images.githubusercontent.com/89059/44516219-65654c00-a6bc-11e8-953d-1a0e19324ded.png">

So, the new label, star ratings, and count with link, are three separate items within a container which needs to centre align items vertically and horizontally, as well as remove the underline from the ratings link.

@ash Do you think that could be covered with two utilities; one to centre align child elements (`u-centerAlign` or similar?), and one to remove the underline from the link?

Reply:
Yeh – I think we should probably have a modifier for links to remove the underline anyway as it seems to be happening a lot across components.  Will probably be good to have this as a modifier or utility so that it's done consistently.

Think having that and the align in a utility makes sense – there's already a type centre helper (may not do exactly what you need if you want to add flex and justify to it as well) > https://github.com/justeat/fozzie/blob/master/src/scss/base/_typography.scss#L224

Reply:
Would it be worth creating something like a `.flex-center` utility, or is that a big can of worms?

Reply:
I remember someone trying that before, and we ended up with it being very granular (with classes for pretty much every flex attribute which was a bit much).

I'm happy for the layout to be contained within each component anyway, simply as it'd be good to port stuff over to using CSS Grid at some point too and that _should_ make it easier to do that bit by bit rather than needing a big bang change.